src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Application/Trucks/Services/ITruckCategoryService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Linq.Expressions;
+
+using Training.TruckWorld.Backend.Domain.Entities;
+
+namespace Training.TruckWorld.Backend.Application.Trucks.Models.Interface;
```

Comment:
folderlarni o'zgartirgan bo'sez namespace larni ham to'g'irlab qo'yasiz

Reply:
namespace edited